dashboard/app/models/user.rb

```diff
@@ -50,10 +50,13 @@
 #  races                    :string(255)
 #  primary_contact_info_id  :integer
 #  unlock_token             :string(255)
+#  cap_state                :string(1)
```

Comment:
Compliance state is defined in: https://github.com/code-dot-org/code-dot-org/blob/032f53af60a78e57580343329769f9226c0197dc/dashboard/lib/policies/child_account.rb#L7

Possible values defined in:
https://github.com/code-dot-org/code-dot-org/blob/778859277f8bdaa46fa4665ac7da2b9b1a2db1b2/lib/cdo/shared_constants.rb#L652-L658

Reply:
I added this note because the term `cap_state` was confusing at first since it relates to US States (as in territories), but we are referring to the status of the compliance. 
I'd consider disambiguating by using the term `cap_status` (`ComplianceStatus`, `CHILD_ACCOUNT_COMPLIANCE_STATUSES`, etc)

Reply:
cc @dayne @wilkie

Reply:
@mgc1194 I agree that it's a little confusing, but we have been consistent about using `us_state` when talking about USA states. I think we should keep it as `_state` because we should feel free to use the word "state" when used in the appropriate context.